### DISCUSSION TITLE
While watching a video file, playback randomly stops and will go to end of video

### DISCUSSION THREAD

**Discussion by killerbeez-yt** [mifi/lossless-cut#1420] [Support]
It depends clip by clip, but sometimes I will be trying to watch a clip all the way through, for example, a 48 second clip. I will watch the clip and halfway through, lets say at the 25 second mark, the clip will stop playing and the playback scrubber will go to the end of the file. I am working with H265 files for what it's worth.

I am on the latest version, 3.48.2.

edit:

here is a video indicating what I am trying to describe https://youtu.be/vJNt3DonHPo

**Comment by mifi** [comment]
Hi. Thanks for reporting. Do you see any errors in the developer console at the time when it jumps to the end? what about older versions like 3.47, does it also happen?
